src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -50,6 +51,18 @@ const EditShiftHours = ({
     setSummaryErrors(errorFields);
   };
 
+  const isFinishTimeNextDay = (
```

Comment:
Simmilar comment here on the function naming. The name suggests a boolean return value, but it's returning a date.